### PR TITLE
Changed hyphen to underscore in chef variable names

### PR DESCRIPTION
**Pull request by bigspotteddog**
For more consistent naming, use underscores instead of hyphens in chef related variable names.

### CODE REVIEW

File: recipes/systemd.rb [hunk 1]
```diff
@@ -1,18 +1,18 @@
 #
-# Cookbook:: nexus-repository-manager
+# Cookbook:: nexus_repository_manager
 # Recipe:: systemd
 #
 # Copyright:: Copyright (c) 2017-present Sonatype, Inc. Apache License, Version 2.0.
 
-systemd_unit 'nexus-repository-manager.service' do
+systemd_unit 'nexus_repository_manager.service' do
```
Comment (whyjustin) [inline recipes/systemd.rb]: Is the service name a chef implementation? I think this may need to be reverted.

Comment (bigspotteddog) [reply recipes/systemd.rb]: Yes, this is a system name and I will revert it.

File: .kitchen.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 driver:
   name: vagrant
   network:
-    - ["private_network", {ip: "192.168.35.35"}]
+    - ["private_network", {ip: "192.168.35.36"}]
```
Comment (whyjustin) [inline .kitchen.yml]: Why this change?

Comment (bigspotteddog) [reply .kitchen.yml]: Changed this so more than one project can be tested at a time. I had two projects using the same ip address and tests failed. Bumped this one to 36.

Comment (whyjustin) [reply .kitchen.yml]: Two projects on the same build agent? If we run into collisions we should start looking into dynamic allocation or else this may get unwieldy.
